docs/docs/external/getting-started/interop/asset-exchange.md

```diff
@@ -55,15 +57,134 @@ run the following steps:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=5 --recipient=alice --locker=bob --target-network=network2 --contract-id=<contract-id> --secret=<hash-pre-image>
        ```
-     - Run the following to verify the claim:
+     - Run the following to trigger `bob`'s claim for `bond01:a03` locked by `alice` in `network1`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=6 --recipient=bob --locker=alice --target-network=network1 --param=bond01:a03 --secret=<hash-pre-image>
        ```
-     - Run the following to trigger `bob`'s claim for `bond01:a03` locked by `alice` in `network1`:
+       
+     Above steps complete a successful asset exchange between two Fabric networks. 
+     In addition to above commands, following are the extra options:
+     - _In case_, `alice` wants to unlock the bond asset, run the following to trigger `alice`'s re-claim for `bond01:a03` locked in `network1`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=7 --locker=alice --recipient=bob --target-network=network1 --param=bond01:a03
        ```
-     - Run the following to verify the claim:
+     - _In case_, `bob` wants to unlock the token asset, run the following to trigger `bob`'s re-claim for `token1:100` locked in `network2`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=8 --locker=bob --recipient=alice --target-network=network2 --contract-id=<contract-id>
        ```
+       
+## Fabric with Corda
+
+We will demonstrate asset exchange of a bond in fabric `network1` with tokens on `Corda_Network`.
+For fabric commands, run from `samples/fabric/fabric-cli` folder, and for Corda commands, run from `samples/corda/corda-simple-application` folder. Here `Alice` and `Bob` in fabric `network1` correspond to `PartyA` (`CORDA_PORT=10006`) and `PartyB` (`CORDA_PORT=10009`) in `Corda_Network` respectively. Following are the step-by-step asset exchange process:
```

Comment:
Capitalize "Fabric"

Reply:
Also, remove the first sentence. I'd prefer you explicitly ask users to navigate to the appropriate folder right before they execute a particular command (just as we have done in other docs) instead of providing a general instruction in the beginning. Retain the second sentence; replace "Here" with "In the subsequent commands", make `alice` and `bob` lower case to correspond to what's specified in below commands.